client/src/components/EditBox.js

```diff
@@ -12,10 +12,26 @@ import {
 
 export default function EditBox() {
   let { id } = useParams();
+  console.log(id);
   const skills = fakeData[id];
   console.log(skills.length);
   const [deleted, setDeleted] = useState(skills);
   const [learningObjective, setLearningObjective] = useState(skills);
+  const [learningObj, setLearningObj] = useState([])
+  useEffect(() => {
+
+    fetch(`/api/learningobjectives/react`)
```

Comment:
Hi @NawalC in response to your question, I think you were saying you want to dynamically load the learning objectives based on the `id` query param? 
```suggestion
    fetch(`/api/learningobjectives/${id}`)
```
Should make that work.